Controller/Api/GroupController.php

```diff
@@ -77,6 +76,20 @@ public function __construct(GroupManagerInterface $groupManager, FormFactoryInte
      */
     public function getGroupsAction(ParamFetcherInterface $paramFetcher)
     {
+        $orderByQueryParam = new QueryParam();
+        $orderByQueryParam->name = 'orderBy';
+        $orderByQueryParam->requirements = 'ASC|DESC';
+        $orderByQueryParam->nullable = true;
+        $orderByQueryParam->strict = true;
+        $orderByQueryParam->description = 'Query groups order by clause (key is field, value is direction)';
+        if (property_exists('FOS\RestBundle\Controller\Annotations\QueryParam', 'map')) {
```

Comment:
> Isn't the following clearer ?

To me both versions are as clear as the other, but this one is more complex (one more variable name).